tests/common/DateField.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import DateField from '../../src/common/DateField';
+
+describe('<DateField />', () => {
```

Comment:
Is it possible to add a test for the returned HTML e.g. the returned className. This is because the DateField file will be marked as untested when you run `yarn coverage`.